tests/test_batch_edit.py

```
@@ -0,0 +1,236 @@
+        """One list_paragraphs() call works for all 10 edits."""
+        doc, _ = multi_para_doc
+        refs = doc.list_paragraphs()
+
+        ops = []
+        for i in range(10):
+            ops.append(EditOperation(
+                action="replace",
+                find=f"item {i + 1}",
+                replace_with=f"BATCH_{i + 1}",
+                paragraph=refs[i].split("|")[0],
+            ))
```

Ruff (0.15.7)

[warning] Use a list comprehension to create a transformed list

(PERF401)